app/src/main/java/com/islamey/hamza/wazaker/utils/Ext.kt

```diff
@@ -0,0 +1,17 @@
+package com.islamey.hamza.wazaker.utils
+
+import android.view.View
+
+object Ext {
+
+    fun View.show() {
+        visibility = View.VISIBLE
+    }
+
+
+    fun View.hide() {
+        visibility = View.GONE
+    }
```

Comment:
we do not need this, as we have ext method called `isVisibile` we can use.

Reply:
> we do not need this, as we have ext method called `isVisibile` we can use.

the `isVisibile` is check if it is visible or not but show make it visiable

Reply:
we have at as setter 
like : 
btnShowZeker. isVisibile = true